### PR TITLE
Initial work for "Wallet Decrypt" component

### PR DESCRIPTION
**Pull request by NessDan**
Fixed up the markup and naming of the plugin, next will be adding state and having it function like it used to.

### CODE REVIEW

**Comment by NessDan** [comment]
I updated the aria-labels as well since they were being used wrong

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

File: common/containers/Tabs/WalletDecrypt/index.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import translate from 'translations';
+
+
+export default class WalletDecrypt extends Component {
+    constructor(props) {
```
Comment (crptm) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: My OCD cant stand this :)

File: common/routing/index.jsx [hunk 2]
```diff
@@ -3,15 +3,15 @@ import {browserHistory, Redirect, Route} from 'react-router';
...
+import WalletDecrypt from 'containers/Tabs/WalletDecrypt'
 import Help from 'containers/Tabs/Help'
 
 export const history = getHistory()
 
 export const Routing = () => (
     <Route name="App" path='' component={App}>
         <Route name="GenerateWallet" path="/" component={GenerateWallet}/>
-        <Route name="ViewWallet" path="/view-wallet" component={ViewWallet}/>
+        <Route name="WalletDecrypt" path="/view-wallet" component={WalletDecrypt}/>
```
Comment (crptm) [inline common/routing/index.jsx]: It is not route tho

File: common/containers/Tabs/WalletDecrypt/index.jsx (hunk 1 above)
Comment (NessDan) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: What exactly?

Comment (crptm) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: Constructor that does nothing :(

File: common/containers/Tabs/WalletDecrypt/index.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                                                aria-labelledby="trezor-label"
+                                                type="radio"
+                                                value="trezor"
+                                                name="142"
+                                            />
+                                            <span id="trezor-label">
+                                                {translate('x_Trezor')}
+                                            </span>
+                                        </label>
+                                        <label className="radio">
```
Comment (dternyak) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: Kinda would rather see all these be an array of dicts defined at the top, and then iterated in here.

NBD though, just a suggestion.

File: common/containers/Tabs/WalletDecrypt/index.jsx (hunk 1 above)
Comment (NessDan) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: Good catch! Haha, removing them now.

File: common/containers/Tabs/WalletDecrypt/index.jsx (hunk 3 above)
Comment (NessDan) [inline common/containers/Tabs/WalletDecrypt/index.jsx]: @dternyak You're absolutely right. I think I will break these out and do that.

I was so fixated on the markup that I forgot to write it right 😢

**Comment by NessDan** [comment]
Closing this, will get more work done to it and hopefully have it even stronger than it was before!

**Pull request closed without merging**